tests/AggregateChangedTest.php

```diff
@@ -28,7 +28,7 @@ public function it_has_a_new_uuid_after_construct()
     {
         $event = AggregateChanged::occur('1', []);
 
-        $this->assertInstanceOf('Rhumsaa\Uuid\Uuid', $event->uuid());
+        $this->assertInstanceOf('Ramsey\Uuid\Uuid', $event->uuid());
```

Comment:
This namespace changes between 2.* and 3.* so you can't simply support "~2.8|~3.0". Can we just get it upgraded?